### PR TITLE
Android: another way of processing mediaButton events.

### PR DESCRIPTION
**Pull request by beqabeqa473**
fixes problem for android 12 #1199

### CODE REVIEW

File: Client/TeamTalkAndroid/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -119,6 +121,8 @@
         </activity>
 
         <receiver android:name=".MediaButtonEventReceiver">
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/AndroidManifest.xml:123]: Hm, this doesn't look right. The tag should not be closed at line 123 I guess?

File: Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java [hunk 2]
```diff
@@ -184,7 +184,7 @@
     AccessibilityAssistant accessibilityAssistant;
     AudioManager audioManager;
     SoundPool audioIcons;
-    ComponentName mediaButtonEventReceiver;
+    //ComponentName mediaButtonEventReceiver;
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java:187]: Delete if not used anymore.

File: Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java [hunk 3]
```diff
@@ -244,7 +244,7 @@ protected void onCreate(Bundle savedInstanceState) {
         restarting = (savedInstanceState != null);
         accessibilityAssistant = new AccessibilityAssistant(this);
         audioManager = (AudioManager) getSystemService(Context.AUDIO_SERVICE);
-        mediaButtonEventReceiver = new ComponentName(getPackageName(), MediaButtonEventReceiver.class.getName());
+        //mediaButtonEventReceiver = new ComponentName(getPackageName(), MediaButtonEventReceiver.class.getName());
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java:247]: Delete if not used anymore.

File: Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java [hunk 4]
```diff
@@ -1763,7 +1763,7 @@ public void onServiceConnected(TeamTalkService service) {
         ttservice.registerUserListener(this);
         ttservice.registerClientListener(this);
         ttservice.setOnVoiceTransmissionToggleListener(this);
-        audioManager.registerMediaButtonEventReceiver(mediaButtonEventReceiver);
+        //audioManager.registerMediaButtonEventReceiver(mediaButtonEventReceiver);
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java:1766]: Delete if not used anymore.

File: Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java [hunk 5]
```diff
@@ -1799,7 +1799,7 @@ public void onServiceConnected(TeamTalkService service) {
     public void onServiceDisconnected(TeamTalkService service) {
         if (wakeLock.isHeld())
             wakeLock.release();
-        audioManager.unregisterMediaButtonEventReceiver(mediaButtonEventReceiver);
+        //audioManager.unregisterMediaButtonEventReceiver(mediaButtonEventReceiver);
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java:1802]: Delete if not used anymore.

**Comment by beqabeqa473** [comment]
All done.

File: Client/TeamTalkAndroid/src/main/AndroidManifest.xml (hunk 1 above)
Comment (bear101) [reply Client/TeamTalkAndroid/src/main/AndroidManifest.xml:123]: <receiver> tag is still not closed correctly

File: Client/TeamTalkAndroid/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -87,6 +87,8 @@
 
         <service
             android:name="dk.bearware.backend.TeamTalkService"
+            android:enabled="true"
+            android:exported="true" />
```
Comment (bear101) [inline Client/TeamTalkAndroid/src/main/AndroidManifest.xml:91]: <service> is double closed.

**Comment by bear101** [comment]
I'm also getting this build error:
```
TeamTalk5/Client/TeamTalkAndroid/src/main/java/dk/bearware/backend/TeamTalkService.java:93: error: package android.support.v4.media does not exist
import android.support.v4.media.MediaMetadataCompat;
```
Did you forget to commit something to build.gradle?

**Comment by beqabeqa473** [comment]
Yes, sorry, I forgot some things in a hurry.

It should be ok for now.

**Pull request merged**